pkg/controller/command/command_controller.go

```diff
@@ -182,27 +202,35 @@ func (r *ReconcileCommand) Reconcile(request reconcile.Request) (reconcile.Resul
 	if err != nil {
 		return reconcile.Result{}, err
 	}
+
 	if err = r.kubernetes.Owner(command).EnsureOwns(psql); err != nil {
 		return reconcile.Result{}, err
 	}
+
 	if !psql.Status.Active {
 		return reconcile.Result{}, nil
 	}
 
-	commandConfigName := command.Name + "-command-configmap"
-	ips := command.Status.IPs
-	if len(ips) == 0 {
-		ips = []string{"0.0.0.0"}
-	}
-	if err = r.configMap(commandConfigName, "command", command, adminPasswordSecret, swiftSecret).ensureCommandConfigExist(ips[0], keystoneAddress, keystonePort, keystoneAuthProtocol, psql.Status.Endpoint); err != nil {
+	config, err := r.getConfig(command)
```

Comment:
Do you check if config is active?

Reply:
Do you ensure that config is owned by command?

Reply:
I wait for Config status to have endpoints. Now ensuring that config is owned by command.